skyportal/services/test_server/test_api_server.py

```diff
@@ -147,6 +147,29 @@ def is_ztf_request(uri):
     assert r1_is_ztf and r2_is_ztf and r1.method == r2.method
 
 
+def kait_request_matcher(r1, r2):
+    """
+    Helper function to help determine if two requests to the KAIT API are equivalent
```

Comment:
Might you need to add some fuzzy matching around the ra, dec position? For instance if two requests have the same filter set but are located a small differences from each other (delta << FOV of KAIT), would that be essentially the same request? Thinking about the case of a refined position of a transient.

Reply:
But this is really for the tests... this is the same as we have done for the others.